inc/saimacsec.h

```diff
@@ -964,30 +964,35 @@ typedef enum _sai_macsec_sa_stat_t
     SAI_MACSEC_SA_STAT_IN_PKTS_LATE,
 
     /**
+     * @brief The sum of this count over Secure Associations gives 802.1ae
```

Comment:
there should be no 2x brief tags, please unify o include only one

Reply:
updated, thanks